test/integration/polymer.d.ts

```diff
@@ -2,6 +2,16 @@
 
 declare namespace Polymer {
   class Element extends HTMLElement {
-    ready():void;
+    ready(): void;
```

Comment:
FYI generated types are in master now: https://github.com/Polymer/polymer/blob/master/types. Not sure how long it will take to get into a release though, so happy to just leave these as-is and switch over to the generated types later.

Reply:
Oh very nice!